content/docs-v1/collection-catalogues.md

```diff
@@ -44,71 +44,76 @@ object_list:
       - id: "cat3b"
 ```
 
-There are two sections in the `objects.yaml` file: `object_list` and `object_display_order`:
+There are two sections in the `objects.yaml` file:  `object_display_order` and `object_list`.
 
-- The `object_list` is a list of the objects and their individual metadata attributes. With the exception of a few reserved terms, as noted in the table below, any attributes can be included here. These attributes and the associated values will ultimately display on the entry pages for each object.
+- You `object_display_order` controls the specifics of which attributes included in your `object_list` should be displayed and in what order. Following the sample above, the attributes included on the pages would be: `artist`, `year`, `dimensions`, `medium`, and `location`.
```

Comment:
"You**r** `object_display_order` ..." instead of "You `object_display_order` ..."